### PR TITLE
add tarball checks

### PR DESCRIPTION
**Pull request by ocefpaf**
@jswhit when I ran this locally I got:

```
> check-manifest --verbose
listing source files under version control: 12 files and directories
building an sdist: cftime-1.1.1.2.tar.gz: 10 files and directories
copying source files to a temporary directory
building a clean sdist: cftime-1.1.1.2.tar.gz: 10 files and directories
lists of files in version control and sdist do not match!
missing from sdist:
  Changelog
  README.release
suggested MANIFEST.in rules:
  include *.release
  include Changelog
```

Do you want to add or skip those? I believe that at least `Changelog` should be there.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -73,4 +77,4 @@ deploy:
     skip_cleanup: true
     on:
       branch: master
-      condition: '${PYTHON_VERSION} == 3.7 && ${BUILD_DOCS} == "true"'
+      condition: '$TRAVIS_JOB_NAME == "tarball"'
```
Comment (ocefpaf) [inline .travis.yml]: It makes sense to deploy only if the tarball tests passed.

File: .travis.yml [hunk 2]
```diff
@@ -43,28 +44,31 @@ install:
   - CYTHON_COVERAGE=1 pip install -e .
 
 script:
-  - if [[ "${BUILD_DOCS}" == "true" ]]; then
+  - if [[ $TRAVIS_JOB_NAME == python-* ]]; then
+      pytest ;
```
Comment (ocefpaf) [inline .travis.yml]: Ideally we should run the tests outside of the build folder. If you are OK with the changes in this commit I'll send a new one to do that too.

**Comment by jswhit** [comment]
I updated MANIFEST.in to include Changelog and *.release in PR #156.  Your suggested changes look good, thanks.

**Comment by jswhit** [comment]
closed by PR #159

**Pull request closed without merging**